frontend/src/styles/mixins.scss

```
@@ -17,13 +17,17 @@ $borderColor_dark: white;
 $gridHighLightColor_light: #e8e8e8;
 $gridHighLightColor_dark: #474747;
 
+$disabledColor_light: #a1a1a1;
```

Stylelint (17.14.1)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)